marketplace-service/src/main/java/com/axonivy/market/repository/impl/CustomProductRepositoryImpl.java

```diff
@@ -57,13 +59,20 @@ private Product findProductById(String id) {
   public Product getProductById(String id) {
     Product result = findProductById(id);
     if (!Objects.isNull(result)) {
```

Comment:
Return early if the product is null
if (Objects.isNull(result)) {
      return null;
}